### PR TITLE
Fstrings

### PR DESCRIPTION
**Pull request by KimaruThagna**
Work done:

Convert any string literals using `.format()` to using `f'{}'` for Python 3 support

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,7 +6,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+.idea/
```
Comment (gmelodie) [inline .gitignore:9]: Isn't this related to another issue?

**Review by gmelodie — APPROVED** [review]
LGTM

**Comment by gleitz** [comment]
This is looking good. Please go ahead and search for all reference to "3.5" in the codebase and remove it since we're dropping support for Python 3.5.

**Comment by gleitz** [comment]
Great work!

**Pull request merged**